Docs/source/dataanalysis/catalyst.rst

```diff
@@ -0,0 +1,162 @@
+.. _visualization-catalyst:
+
+In situ Visualization with Catalyst 2
+=====================================
+Catalyst 2 (further referred to as just Catalyst) is a lightweight in-situ visualization and analysis framework API developed for simulations and other scientific data producers. It has a lightweight implementation
+(or **stub**) and an SDK to develop custom implementations of Catalyst. ParaView comes with its own implementation (known as **ParaView Catalyst**) for leveraging ParaView's
+visualization and analysis capabilities, which is what this document will focus on.
+
+
+Enabling Catalyst
+-----------------
+In order to use Catalyst with WarpX, you must `build Catalyst 2 <https://catalyst-in-situ.readthedocs.io/en/latest/build_and_install.html>`_ and `build <https://github.com/Kitware/ParaView/blob/master/Documentation/dev/build.md>`_ or `install <https://www.paraview.org/download/>`_ ParaView 5.9+. Afterward, AMReX must be built with ``AMReX_CONDUIT=TRUE``,
```

Comment:
Anonymous links for the general keywords (build, install) to avoid collisions in the sphinx reference of links:
```suggestion
In order to use Catalyst with WarpX, you must `build Catalyst 2 <https://catalyst-in-situ.readthedocs.io/en/latest/build_and_install.html>`_ and `build <https://github.com/Kitware/ParaView/blob/master/Documentation/dev/build.md>`__ or `install <https://www.paraview.org/download/>`__ ParaView 5.9+. Afterward, AMReX must be built with ``AMReX_CONDUIT=TRUE``,
```